Description: Drop git from sphinx config since git repo is not available
             during package builds.
Author: Corey Bryant <corey.bryant@canonical.com>
Forwarded: not-needed

--- a/doc/source/conf.py
+++ b/doc/source/conf.py
@@ -51,10 +51,10 @@
 
 modindex_common_prefix = ['glance_store.']
 
-git_cmd = ["git", "log", "--pretty=format:'%ad, commit %h'", "--date=local",
-           "-n1"]
-html_last_updated_fmt = subprocess.Popen(git_cmd,
-                                         stdout=subprocess.PIPE).communicate()[0]
+#git_cmd = ["git", "log", "--pretty=format:'%ad, commit %h'", "--date=local",
+#           "-n1"]
+#html_last_updated_fmt = subprocess.Popen(git_cmd,
+#                                         stdout=subprocess.PIPE).communicate()[0]
 
 
 # Grouping the document tree into LaTeX files. List of tuples
